layout-management-services/pom.xml

```diff
@@ -27,12 +27,87 @@
     <version>1.5.x-experience-SNAPSHOT</version>
   </parent>
   <artifactId>layout-management-services</artifactId>
+  <packaging>jar</packaging>
   <name>eXo Add-on:: Layout management - Services</name>
+  <description>Layout Management Services</description>
+  <properties>
+    <rest.api.doc.title>Layout Management Rest APIs</rest.api.doc.title>
+    <rest.api.doc.version>1.0</rest.api.doc.version>
+    <rest.api.doc.description>Layout Management addon used Rest endpoints</rest.api.doc.description>
+  </properties>
   <dependencies>
+    <dependency>
+      <groupId>${project.groupId}</groupId>
+      <artifactId>layout-management-api</artifactId>
+      <version>${project.version}</version>
+      <scope>provided</scope>
+    </dependency>
     <dependency>
       <groupId>javax.servlet</groupId>
       <artifactId>javax.servlet-api</artifactId>
       <scope>provided</scope>
     </dependency>
+      <dependency>
+          <groupId>org.exoplatform.ws</groupId>
+          <artifactId>exo.ws.rest.core</artifactId>
+          <version>6.5.x-maintenance-SNAPSHOT</version>
+          <scope>compile</scope>
+      </dependency>
+    <dependency>
+      <groupId>io.swagger.core.v3</groupId>
+      <artifactId>swagger-annotations</artifactId>
+    </dependency>
+    <dependency>
+      <groupId>org.exoplatform.commons</groupId>
+      <artifactId>commons-component-common</artifactId>
+    </dependency>
   </dependencies>
+  <build>
+    <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-surefire-plugin</artifactId>
+        <configuration>
+          <argLine>@{argLine} ${env.MAVEN_OPTS} --add-opens=java.base/java.security=ALL-UNNAMED   --add-opens=java.base/jdk.internal.loader=ALL-UNNAMED  --add-opens=java.base/java.text=ALL-UNNAMED --add-opens=java.base/java.util=ALL-UNNAMED  --add-opens=java.base/java.util.regex=ALL-UNNAMED   --add-opens=java.base/java.lang=ALL-UNNAMED  --add-opens=java.base/java.util.concurrent=ALL-UNNAMED </argLine>
+        </configuration>
+      </plugin>
+      <plugin>
+        <groupId>org.projectlombok</groupId>
+        <artifactId>lombok-maven-plugin</artifactId>
+        <executions>
+          <execution>
+            <phase>generate-sources</phase>
+            <goals>
+              <goal>delombok</goal>
+            </goals>
+            <configuration>
+              <sourceDirectory>${basedir}/src/main/java</sourceDirectory>
+              <addOutputDirectory>false</addOutputDirectory>
+            </configuration>
+          </execution>
+        </executions>
+      </plugin>
```

Comment:
To start we add only needed plugins, remove the unneeded one then we can add others if needed